gettsim/transfers/rente.py

```diff
@@ -296,3 +319,303 @@ def ges_rente_regelaltersgrenze(geburtsjahr: int, ges_rente_params: dict) -> flo
         ],
     )
     return out
+
+
+def ges_rente_grenze_langjährig_frau(
+    geburtsjahr: int,
+    geburtsmonat: int,
+    ges_rente_params: dict,
+    ges_rente_vorraussetz_frauen: int,
+    ges_rente_vorraussetz_langjährig: int,
+) -> float:
+    """Calculates the age, at which a longterm insured person or women is eligible to
+        claim the full pension. Full retirement age (FRA) without deductions.
+        This age is smaller or equal to the regelaltersgrenze (FRA<=NRA).
+        These thresholds are relevant for the size of the Zugangsfaktor.
+
+
+    Parameters
+    ----------
+    geburtsjahr
+        See basic input variable :ref:`geburtsjahr <geburtsjahr>`.
+    geburtsmonat
+        See basic input variable :ref:`geburtsmonat <geburtsmonat>`.
+    ges_rente_params
+        See params documentation :ref:`ges_rente_params <ges_rente_params>`.
+    ges_rente_vorraussetz_frauen
+        See :func:`ges_rente_vorraussetz_frauen`
+    ges_rente_vorraussetz_langjährig
+        See :func:``ges_rente_vorraussetz_langjährig``
+
+    Returns
+    -------
+
+    """
+    elig_long = ges_rente_vorraussetz_langjährig
+    elig_women = ges_rente_vorraussetz_frauen
+
+    if geburtsjahr < 1951:
+        x_long = geburtsjahr + (geburtsmonat - 1) / 12
+    else:
+        x_long = geburtsjahr
+
+    if elig_long == 1:
+        pension_for_long = piecewise_polynomial(
+            x=x_long,
+            thresholds=ges_rente_params["altersgrenze_langjährig_versicherte"][
+                "thresholds"
+            ],
+            rates=ges_rente_params["altersgrenze_langjährig_versicherte"]["rates"],
+            intercepts_at_lower_thresholds=ges_rente_params[
+                "altersgrenze_langjährig_versicherte"
+            ]["intercepts_at_lower_thresholds"],
+        )
+    else:
+        pension_for_long = 9000
```

Comment:
Why set to this value? At least explain in comment.

Reply:
done